saleor/graphql/api.py

```diff
@@ -28,6 +28,16 @@ def get_ancestors_from_cache(category, context):
         return category.get_ancestors()
 
 
+class TranslatedModel(object):
```

Comment:
But why is it a mixin instead of a subclass of `DjangoObjectType`? And why does it have to check whether a node has a `translated` attribute?